rero_ils/modules/stats/jsonschemas/stats/stat-v0.0.1.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Stats",
+  "description": "JSON schema for statistics.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "values"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate the records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://bib.rero.ch/schemas/stats/stat-v0.0.1.json"
+    },
+    "pid": {
+      "title": "ItemType ID.",
+      "type": "string",
+      "minLength": 1
+    },
+    "values": {
+      "title": "Statistics",
```

Comment:
Why `values` and not `stats` or ` statistics`?

Reply:
I can but basically stats is the root.

Reply:
ok.